registries.conf

```diff
@@ -37,14 +40,18 @@
 #
 # # The physical location of the "prefix"-rooted namespace.
 # #
-# # By default, this equal to "prefix" (in which case "prefix" can be omitted
+# # By default, this is equal to "prefix" (in which case "prefix" can be omitted
 # # and the [[registry]] TOML table can only specify "location").
 # #
 # # Example: Given
 # #   prefix = "example.com/foo"
 # #   location = "internal-registry-for-example.net/bar"
 # # requests for the image example.com/foo/myimage:latest will actually work with the
 # # internal-registry-for-example.net/bar/myimage:latest image.
+#
+# # The location can be empty iff prefix is in a
```

Comment:
Note: I think we should support the same behavior for ordinary prefixes (in the future).

Reply:
should this note be included in the doc?

Reply:
Apologies for being vague. It was just a general comment, so no need to put that in the doc.

Reply:
AFAICS the implementation in this PR does support empty location for non-wildcards.

As for documenting this as officially supported, I agree it would be nice, for the future — I find “prefix is for matching, location is for redirection” much easier to explain than the current “location must always be set, and it is used for matching if prefix is not present”. Probably together with finalizing the empty value validation.

Reply:
hmm, i just now added a check for location being empty only for wildcarded prefix. Let me know if I should undo it :)